app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -552,6 +552,13 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = false,
     )
 
+    var lastInteractionWithReEngageCookieBannerDialogInMS by longPreference(
+        appContext.getPreferenceKey(
+            R.string.pref_key_cookie_banner_re_engage_dialog_last_interaction_with_dialog_in_ms,
```

Comment:
This seems like a long string name and dialog is already stated so I think this may be better:

```suggestion
            R.string.pref_key_cookie_banner_re_engage_dialog_last_interaction_in_ms,
```

Reply:
Thanks @zmckenney, I pushed an extra commit to address the suggestions :)